doc/whats_new/v1.0.rst

```diff
@@ -575,13 +575,19 @@ Changelog
 - |Fix| Fixed a bug in :func:`utils.sparsefuncs.mean_variance_axis` where the
   precision of the computed variance was very poor when the real variance is
   exactly zero. :pr:`19766` by :user:`Jérémie du Boisberranger <jeremiedbb>`.
-
+  
 :mod:`sklearn.validation`
 .........................
 
 - |Fix| Support for `np.matrix` is deprecated in
   :func:`~sklearn.utils.check_array` in 1.0 and will raise a `TypeError` in
   1.2. :pr:`20165` by `Thomas Fan`_.
+  
+:mod: `sklearn.metrics`
+.......................
+
+- |Feature|  :func:`~sklearn.metrics.mean_squared_log_error` now supports
+  `squared='False'`. :pr:`20326` by :user:`Uttam kumar <helper-uttam>`_.
```

Comment:
The doc CI is failing because of those two warnings:

```
doc/whats_new/v1.0.rst:587: WARNING: Field list ends without a blank line; unexpected unindent.
doc/whats_new/v1.0.rst:589: WARNING: Mismatch: both interpreted text role prefix and reference suffix. 
```

The error messages are weird but I suspect this might be caused because there is already an section named `sklearn.metrics` upper in the same file. The module sections are ordered by alphabetical order.

Reply:
Ops! I thought it was occurring due to some whitespaces or extra lines. :)

Reply:
Please move this entry to the `sklearn.metrics` on line 410 in the same file to avoid redefining the `sklearn.metrics` section.

Reply:
Thanks for the suggestion, I didn't noticed this previously!

I had moved to 410.